contrib/julia.xml

```diff
@@ -198,7 +196,6 @@
       <item> UInt128 </item>
       <item> Union </item>
       <item> Unsigned </item>
-      <item> UTF8String </item>
```

Comment:
this file also still has `CharString` around, and not referring to Stefan's new test version, but to the 0.2-and-older name `UTF32String` used to have